lib/rdf/util/coercions.rb

```diff
@@ -0,0 +1,52 @@
+module RDF
+  module Util
+    module Coercions
+      # This is a provisional module intended to house input
+      # coercions. Currently the only coercion is a statement
+      # preprocessor that is used in e.g. {RDF::Writable#insert} and
+      # {RDF::Mutable#delete}.
+
+      ##
+      # Coerce a set of arguments into {RDF::Statement} objects and then
+      # operate over them with a block.
```

Comment:
"operate over them" seems a bit generic, when what it actually seems to do is to coerce the input to statements, and yield an iterator over those coerced statements.

Reply:
The original code handed off to either `Writable#insert_statements` or `Mutable#delete_statements`, so I guess what it actually expects is an `RDF::Enumerable` or one or more `RDF::Statement` objects.